force-app/main/default/objects/Case/fields/Rationale__c.field-meta.xml

```diff
@@ -5,7 +5,7 @@
     <label>Rationale</label>
     <length>32768</length>
     <trackFeedHistory>false</trackFeedHistory>
-    <trackHistory>false</trackHistory>
```

Comment:
Any particular reason to remove tracking history that was there earlier.

Reply:
Hi Sunil, here we are enabling the field tracking that was not there earlier as per user story ESA-1316.